### ISSUE TITLE
[Bug]: Sample images for training LoRA are broken

### ISSUE THREAD

**Issue by ashleykleynhans** [d8ahazard/sd_dreambooth_extension#1273]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

When enabling sample images during training, the sample images just display noise and don't display useful samples.
<img width="297" alt="Screenshot 2023-06-09 at 19 44 36" src="https://github.com/d8ahazard/sd_dreambooth_extension/assets/886709/498aa7f3-ad9f-4ce1-a45b-c0e30826bfc5">

### Steps to reproduce the problem

1. Configure Dreambooth extension for training a LoRA.
2. Enable saving of sample images during training.
3. Observe that the sample images are not generated correctly.

### Commit and libraries

Initializing Dreambooth
Dreambooth revision: dc413a14379b165355502d9f65856c40a4bb5b6f
[+] xformers version 0.0.20 installed.
[+] torch version 2.0.1+cu118 installed.
[+] torchvision version 0.15.2+cu118 installed.
[+] accelerate version 0.19.0 installed.
[+] diffusers version 0.16.1 installed.
[+] transformers version 4.30.0 installed.
[+] bitsandbytes version 0.35.4 installed.

### Command Line Arguments

```Shell
--listen
  --no-hashing \
  --skip-python-version-check \
  --skip-torch-cuda-test \
  --enable-insecure-extension-access \
  --api \
  --skip-version-check \
  --skip-install
```

### Console logs

```Shell
N/A, no errors in the console, the images just aren't generated correctly.
```

### Additional information

This issue is specific to LoRA training.  The sample images are generated correctly for full Dreambooth training.

**Comment by morphinapg** [comment]
Same issue here. Once I hit cancel training, I'm able to generate samples in the generate tab, but they look identical to the base model I'm training based on, confirmed by generating ckpt and comparing with the base model in txt2img. I'm trying to use the new AdamW D-Adaptation optimizer though, so maybe that has something to do with it, even though the learning rates created by that do seem fairly high though, so I would think I'd see some results.

**Comment by zono50** [comment]
I've tried multiple optimizers, even did 8 bit adam, and same thing. Scrambled preview images

**Comment by mrlzla** [comment]
the same issue

**Comment by Dwuuds** [comment]
same issue here

**Comment by xexzeed** [comment]
i've spent some time trying to figure this one out, Everything broke when automatics was updated before that dreambooth was working wonderfully, i've tried the dev branch and that commit and cant seem to get back to a functioning state, the dev branch doesn't give errors but generates these noise images of almost solid color, same for the main branch and what ever is downloaded threw the automatic1111 plugin manager, 926ae204ef5de17efca2059c334b6098492a0641 generates good sample images but if i x plot the same seed across the checkpoints nothing is actually changing. The real confusion begins when i reverted to the version of automatic1111 that i had it all working on and the problems started cropping up there

**Comment by Dachannien** [comment]
Same issue here.

**Comment by LiteningCat** [comment]
I’m not getting anything anymore just my loss and RAM
![image](https://github.com/d8ahazard/sd_dreambooth_extension/assets/134942236/2354b14f-b319-43c4-88ef-8bcf4ef20f6f)

**Comment by ChunkySmoove** [comment]
still having this issue as well

**Comment by sailxjx** [comment]
some here

**Comment by mhvelplund** [comment]
Everyone having this issue, upvote the issue instead of writing "me too".

**Comment by mayur-rank** [comment]
Same here

**Comment by Neltherion** [comment]
I also have this issue, how can I upvote it?

**Comment by joneschunghk** [comment]
Same problem, although I followed the recommended settings on the release notes

**Comment by morphinapg** [comment]
Has anybody tested the PR that claims to fix this?

https://github.com/d8ahazard/sd_dreambooth_extension/pull/1336

Also curious if it fixes sample generation when not using Lora as well.

**Comment by ignis-sec** [comment]
@morphinapg, I didn't know non-Lora training sample generation was broken. Changes in that PR are specifically for fixing trainable lora injection when reconstructing the pipeline for generation, so it shouldn't have an effect for non-lora training.

**Comment by Neltherion** [comment]
Was this fixed? 🤔

**Comment by rokonezoko55555555** [comment]
same issue but i cant even see imgs
